tests/operators/hive_operator.py

```diff
@@ -91,6 +91,16 @@ def test_hiveconf_jinja_translate(self):
         t.prepare_template()
         self.assertEqual(t.hql, "SELECT {{ num_col }} FROM {{ table }};")
 
+    def test_hiveconf(self):
+        hql = "SELECT * FROM ${hiveconf:table} PARTITION (${hiveconf:day});"
+        t = operators.hive_operator.HiveOperator(
+            hiveconfs={'table': 'static_babynames', 'day': '{{ ds }}'},
+            task_id='dry_run_basic_hql', hql=hql, dag=self.dag)
+        t.prepare_template()
+        self.assertEqual(
+            t.hql,
+            "SELECT * FROM ${hiveconf:table} PARTITION (${hiveconf:day});")
```

Comment:
The assert should be done against the templates version, right? So:
```
"SELECT * FROM static_babynames PARTITION ({{ ds }});"
```

Reply:
The test is to show that hql is not changed after templating as they need to preserve the original `${hiveconf:var}` format for `--hiveconf key=value` params.